### PR TITLE
Add integration tests example and usage within emulator

### PR DESCRIPTION
**Pull request by m-Peter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +110,20 @@ Test results: "test_array_utils.cdc"
 - PASS: testMap
 - PASS: testMapStrings
 - PASS: testReduce
+Coverage: 90.6% of statements
+```
+
+Look at the files `ArrayUtils.cdc` (smart contract) and `test_array_utils.cdc` (tests for the smart contract). For the `ArrayUtils.range` method, we have omitted in purpose the branch where `start > end`. It is left as an exercise to the reader. Look at the comment on line 25 in `test_array_utils.cdc`.
```
Comment (turbolent) [inline README.md]:
```suggestion
Look at the files `ArrayUtils.cdc` (smart contract) and `test_array_utils.cdc` (tests for the smart contract). 
For the `ArrayUtils.range` method, we have omitted the code branch where `start > end` on purpose. It is left as an exercise for the reader. Look at the comment on line 25 in `test_array_utils.cdc`.
```

File: README.md [hunk 2]
```diff
@@ -117,9 +138,56 @@ Test results: "test_string_utils.cdc"
...
+The generated `coverage.json` file is somewhat more elaborate, for integration tests. By viewing its content, we find the following keys:
+
+- `A.01cf0e2f2f715450.ArrayUtils`
+- `A.01cf0e2f2f715450.StringUtils`
+- `A.0ae53cb6e3f42a79.FlowToken`
+- `A.e5a8b7f23e8b548f.FlowFees`
+- `A.ee82856bf20e2aa6.FungibleToken`
+- `A.f8d6e0586b0a20c7.FlowServiceAccount`
+
+and some other locations. The ones starting with `s.*` are scripts, while the ones starting with `t.*` are transactions.
```
Comment (turbolent) [inline README.md]:
```suggestion
and some other locations.

Locations that start with `A.` are contracts deployed to an account, ones that start with `s.` are scripts, and ones that start with `t.` are transactions.
```

File: README.md [hunk 3]
```diff
@@ -117,9 +138,56 @@ Test results: "test_string_utils.cdc"
...
+```bash
+curl -XPUT 'http://localhost:8080/emulator/codeCoverage/reset'
+```
+
+Which results in the following:
+
+![Code Coverage Reset](./code-coverage-reset.png)
 
-Also, look at the files `StringUtils.cdc` (smart contract) and `test_string_utils.cdc` (tests for the smart contract).
+All of the keys have disappeared, except for the `A.f8d6e0586b0a20c7.FlowServiceAccount`, which is a system contract that runs frequently some operations.
```
Comment (turbolent) [inline README.md]:
```suggestion
All of the keys have disappeared, except for `A.f8d6e0586b0a20c7.FlowServiceAccount`, which is a system contract that is essential to the operations of Flow.
```

**Review by turbolent — COMMENTED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (m-Peter) [reply README.md]: Updated in https://github.com/Build-Squad/flow-code-coverage/pull/1/commits/4fd71019aea467b88db1ed29592dc79b2eca7176

File: README.md (hunk 2 above)
Comment (m-Peter) [reply README.md]: Updated in https://github.com/Build-Squad/flow-code-coverage/pull/1/commits/d61f8dbce7558878cf4b518c144045c7d91f292b

File: README.md (hunk 3 above)
Comment (m-Peter) [reply README.md]: Updated in https://github.com/Build-Squad/flow-code-coverage/pull/1/commits/385067c991608e7a37eba742bd857fab136ebeb0

**Review by turbolent — APPROVED** [review]
Nice!

**Pull request merged**